cartridges/int_styla/cartridge/scripts/StylaMain.js

```diff
@@ -131,247 +128,267 @@ function getContentVersion(config) {
-		idx = path.indexOf(basePath);
-		if (idx > -1) {
-			result = path.substr(0, idx + basePath.length);
-		}
-	}
-	return result;
+        idx = path.indexOf(basePath);
+        if (idx > -1) {
+            result = path.substr(0, idx + basePath.length);
+        }
+    }
+    return result;
 }
 
 
 /**
  * Read magazine configuration from custom object.
- * 
+ *
  * @param obj 				Custom object which stores a magazine configuration.
  * @param path 				Path of the request for which the magazine will be displayed.
  * @param getContentVer		If true, then the request to get the content version is performed.
  * 							If false, then the result object's @jsLibUrl and @cssUrl maybe invalid and shouldn't be used.
- * 
+ *
  * @returns object
  */
 function getMagazineConfigurationFromObj(obj, path, getContentVer) {
-	var site = Site.getCurrent(),
-		cdnBaseUrl = site.getCustomPreferenceValue('stylaCdnBaseUrl'),
-		cssBaseUrl = site.getCustomPreferenceValue('stylaCssBaseUrl'),
-		version = '',
-		versionResponse,
-		config = {
-			valid        : true,
-			path         : path,
-			rootPath     : null,
-			username     : obj.custom.username   || '',
-			pipeline     : obj.custom.pipeline   || '',
-			categoryID   : obj.custom.categoryID || '',
-			basePath     : obj.custom.basePath   || '',
-			seoDisabled  : site.getCustomPreferenceValue('stylaSeoDisabled'),
-			seoCachingTTL: site.getCustomPreferenceValue('stylaSeoCachingTTL'),
-			jsLibUrl     : cdnBaseUrl.replace('{USER}', obj.custom.username || ''),
-			cssUrl       : cssBaseUrl.replace('{USER}', obj.custom.username || '')
-		};
-
-	if (getContentVer) {
-		versionResponse = getContentVersion(config);
-		if (!versionResponse.error && !empty(versionResponse.version)) {
-			version = versionResponse.version;
-		}
-	}
-
-	config.jsLibUrl = config.jsLibUrl.replace('{VERSION}', version);
-	config.cssUrl   = config.cssUrl.replace('{VERSION}', version);
-
-	if (!config.seoDisabled) {
-		config.seoResponse = getSeoContent(config);
-	}
-	
-	config.rootPath = getMagazineRootPath(config.path, config.basePath);
-	
-	return config;
+    var site = Site.getCurrent(),
+        cdnBaseUrl = site.getCustomPreferenceValue('stylaCdnBaseUrl'),
+        cssBaseUrl = site.getCustomPreferenceValue('stylaCssBaseUrl'),
+        version = '',
+        versionResponse,
+        config = {
+            valid: true,
+            path: path,
+            rootPath: null,
+            username: obj.custom.username || '',
+            pipeline: obj.custom.pipeline || '',
+            categoryID: obj.custom.categoryID || '',
+            basePath: obj.custom.basePath || '',
+            seoDisabled: site.getCustomPreferenceValue('stylaSeoDisabled'),
+            seoCachingTTL: site.getCustomPreferenceValue('stylaSeoCachingTTL'),
+            jsLibUrl: cdnBaseUrl.replace('{USER}', obj.custom.username || ''),
+            cssUrl: cssBaseUrl.replace('{USER}', obj.custom.username || '')
+        };
+
+    if (getContentVer) {

```

Comment:
the whole version logic can be removed, the `?v=` parameter doesn't need to be appended anymore therefor we can save a request here and remove the whole version logic and remove the placeholder from the initial javascript and css script and link tags ...